Automattic/wp-calypso
# Add Jetpack Starter benefits list

## Proposed Changes

* This PR adds all the necessary constants, helper functions, and references to support the new Jetpack Starter plan. The focus of the PR is to include the benefits list on the checkout page.

## Testing Instructions



* Checkout this branch locally
* Visit http://calypso.localhost:3000/checkout/jetpack/jetpack_starter_yearly and http://calypso.localhost:3000/checkout/jetpack/jetpack_starter_monthly
* Make sure you see a list of features under the "Included with your purchase" section
* Confirm that the list only shows Backup and Anti-spam features and the copy looks right
* Make sure the features list looks like the screenshot below

<img width="351" alt="image" src="https://user-images.githubusercontent.com/5714212/236083971-331731da-0b3a-4a56-95cd-b5c68fbf7513.png">



## Pre-merge Checklist
